amaranth/vendor/__init__.py

```diff
@@ -30,6 +31,9 @@ def __getattr__(name):
     if name == "GowinPlatform":
         from ._gowin import GowinPlatform
         return GowinPlatform
+    if name == "GateMatePlatform":
+        from ._gatemate import GateMatePlatform
+        return GateMatePlatform
```

Comment:
`Ga...` should be sorted above `Go...`.

Reply:
Placed `GatematePlatform` above `GowinPlatform`